Update printoperator.md (#1262)
MicrosoftDocs/dataexplorer-docs: data-explorer/kusto/query/printoperator.md

---
# print operator

> Outputs single-row with one or more scalar expression results as columns.

Reviewer: ```suggestion
Outputs a single-row with one or more scalar expression results as columns.
```

## Returns

> A one or more column, single-row, table whose single cell has the value of the evaluated *ScalarExpression*.

Reviewer: Needs rephrasing, it's not a "single cell" anymore :)
By the way, the beginning of the sentence is "A one or more column" sounds a bit strange (though grammatically it's probably correct).

> A one or more column, single-row, table whose single cell has the value of the evaluated *ScalarExpression*.

Reviewer 2: ```suggestion
A table with one or more columns and a single row, with the value of the evaluated *ScalarExpression*.
```
